plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -98,7 +98,9 @@ class ContainerServiceFormSection extends Component {
     const inputFiled = (
       <FieldInput
         key="gpus-input"
+        min="0.001"
```

Comment:
This will cause the value to default to 0, which strikes me as a bad value for number of cpus...